src/lib/containers/HasAccess.tsx

```diff
@@ -321,30 +323,19 @@ export default class HasAccess extends React.Component<
       linkText = 'View Terms'
     }
     return (
-      <>
```

Comment:
Could you revert this, putting the button in instead of the anchor? Sorry I must've not been clear on what I said with merging these two.